src/Nethermind/Nethermind.Dsl.Test/DslRpcModuleTests.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Threading.Tasks;
+using Nethermind.Api;
+using Nethermind.Api.Extensions;
+using Nethermind.Blockchain.Processing;
+using Nethermind.Core;
+using Nethermind.Core.Crypto;
+using Nethermind.Core.Extensions;
+using Nethermind.Core.Test.Builders;
+using Nethermind.Int256;
+using Nethermind.Serialization.Json;
+using NSubstitute;
+using NUnit.Framework;
+
+namespace Nethermind.Dsl.Test
+{
+    public class DslRpcModuleTests
+    {
+        private DslPlugin _dslPlugin;
+        private INethermindApi _api;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _dslPlugin = new DslPlugin();
+            _api = Substitute.For<INethermindApi>();
+            _dslRpcModule = new DslRpcModule();
```

Comment:
this is not up to date with current DSL branch, right now DslRpcModule takes api, logger and collection of interpreters from plugin (all of them will need to be mocked here)